pkg/anypoint/client.go

```diff
@@ -335,7 +337,7 @@ func (c *AnypointClient) GetExchangeFileContent(link, packaging, mainFile string
 
 // GetAnalyticsWindow lists the managed assets in Mulesoft: https://docs.qax.mulesoft.com/api-manager/2.x/analytics-event-api
 func (c *AnypointClient) GetAnalyticsWindow() ([]AnalyticsEvent, error) {
-	startDate, endDate := c.getLastRun()
```

Comment:
The start and end dates can be received as params.

Reply:
👍